src/main/fc/rc_controls.h

```diff
@@ -74,7 +74,8 @@ typedef enum {
 typedef enum {
     RC_SMOOTHING_DERIVATIVE_OFF,
     RC_SMOOTHING_DERIVATIVE_PT1,
-    RC_SMOOTHING_DERIVATIVE_BIQUAD
+    RC_SMOOTHING_DERIVATIVE_BIQUAD,
+    RC_SMOOTHING_DERIVATIVE_AUTO
```

Comment:
A comma after the last value will make the changeset look much cleaner when the next value is added.

Reply:
fixed.